[SQS] ARN as queue name results in No enum constant com.amazonaws.regions.Regions.us-west-1

## This is a...

<pre><code>
[ ] Feature request
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  <!-- Please search GitHub for a similar issue or PR before submitting -->
[ ] Documentation issue or request
</code></pre>

## Description
The ARN of my queue is `arn:aws:sqs:us-west-1:928544467084:in` and 
when I use it as the queue name, the integration startup fails with:

```
2019-04-26 12:22:40.788  INFO 1 --- [           main] i.s.i.c.proxy.ComponentProxyComponent    : Connector resolved: aws-sqs-0-0 -> aws-sqs://arn:aws:sqs:us-west-1:928544467084:in?accessKey=RAW(<key here>)&delay=500&deleteAfterRead=true&deleteIfFiltered=true&maxMessagesPerPoll=1&region=US_WEST_1&secretKey=xxxxxx
2019-04-26 12:22:40.942  INFO 1 --- [           main] i.s.i.c.proxy.ComponentProxyComponent    : Connector resolved: aws-sqs-0-1 -> aws-sqs://arn:aws:sqs:us-west-1:928544467084:out?accessKey=RAW(<key here>)&delaySeconds=0&messageDeduplicationIdStrategy=useExchangeId&messageGroupIdStrategy=useConstant&region=US_WEST_1&secretKey=xxxxxx
2019-04-26 12:22:40.991  INFO 1 --- [           main] o.a.c.impl.DefaultStreamCachingStrategy  : StreamCaching in use with spool directory: /tmp/camel/camel-tmp-ddc47350-ff7a-42d9-aaa8-6468cd8fd438 and rules: [Spool > 128K body size]
2019-04-26 12:22:41.035  INFO 1 --- [           main] o.a.camel.spring.SpringCamelContext      : Apache Camel 2.21.0.fuse-740005 (CamelContext: sqs) is shutting down
2019-04-26 12:22:41.058 DEBUG 1 --- [           main] i.s.i.c.proxy.ComponentProxyComponent    : Stopping connector: aws-sqs-0-1
2019-04-26 12:22:41.058 DEBUG 1 --- [           main] i.s.i.c.proxy.ComponentProxyComponent    : Stopping connector: aws-sqs-0-0
2019-04-26 12:22:41.073  INFO 1 --- [           main] o.a.camel.spring.SpringCamelContext      : Apache Camel 2.21.0.fuse-740005 (CamelContext: sqs) uptime 0.607 seconds
2019-04-26 12:22:41.076  INFO 1 --- [           main] o.a.camel.spring.SpringCamelContext      : Apache Camel 2.21.0.fuse-740005 (CamelContext: sqs) is shutdown in 0.041 seconds
2019-04-26 12:22:41.090  INFO 1 --- [           main] utoConfigurationReportLoggingInitializer : 
Error starting ApplicationContext. To display the auto-configuration report re-run your application with 'debug' enabled.
2019-04-26 12:22:41.120 ERROR 1 --- [           main] o.s.boot.SpringApplication               : Application startup failed
org.apache.camel.RuntimeCamelException: org.apache.camel.FailedToCreateRouteException: Failed to create route -LdOZtsXN9q6Eay-U8rp: Route(-LdOZtsXN9q6Eay-U8rp)[[From[aws-sqs-0-0]] -> [SetHeade... because of No enum constant com.amazonaws.regions.Regions.us-west-1
	at org.apache.camel.util.ObjectHelper.wrapRuntimeCamelException(ObjectHelper.java:1830) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.spring.SpringCamelContext.start(SpringCamelContext.java:136) ~[camel-spring-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.spring.SpringCamelContext.onApplicationEvent(SpringCamelContext.java:174) ~[camel-spring-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.springframework.context.event.SimpleApplicationEventMulticaster.doInvokeListener(SimpleApplicationEventMulticaster.java:172) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.event.SimpleApplicationEventMulticaster.invokeListener(SimpleApplicationEventMulticaster.java:165) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.event.SimpleApplicationEventMulticaster.multicastEvent(SimpleApplicationEventMulticaster.java:139) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:393) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.publishEvent(AbstractApplicationContext.java:347) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.finishRefresh(AbstractApplicationContext.java:883) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.finishRefresh(EmbeddedWebApplicationContext.java:144) ~[spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:546) ~[spring-context-4.3.19.RELEASE.jar!/:4.3.19.RELEASE]
	at org.springframework.boot.context.embedded.EmbeddedWebApplicationContext.refresh(EmbeddedWebApplicationContext.java:122) ~[spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:693) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:360) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:303) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1118) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1107) [spring-boot-1.5.16.RELEASE.jar!/:1.5.16.RELEASE]
	at io.syndesis.example.Application.main(Application.java:13) [classes!/:na]
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[na:1.8.0_191]
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[na:1.8.0_191]
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[na:1.8.0_191]
	at java.lang.reflect.Method.invoke(Method.java:498) ~[na:1.8.0_191]
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:48) [project-0.1-SNAPSHOT.jar:na]
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:87) [project-0.1-SNAPSHOT.jar:na]
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:50) [project-0.1-SNAPSHOT.jar:na]
	at org.springframework.boot.loader.PropertiesLauncher.main(PropertiesLauncher.java:595) [project-0.1-SNAPSHOT.jar:na]
Caused by: org.apache.camel.FailedToCreateRouteException: Failed to create route -LdOZtsXN9q6Eay-U8rp: Route(-LdOZtsXN9q6Eay-U8rp)[[From[aws-sqs-0-0]] -> [SetHeade... because of No enum constant com.amazonaws.regions.Regions.us-west-1
	at org.apache.camel.impl.RouteService.warmUp(RouteService.java:147) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.impl.DefaultCamelContext.doWarmUpRoutes(DefaultCamelContext.java:3947) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.impl.DefaultCamelContext.safelyStartRouteServices(DefaultCamelContext.java:3854) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.impl.DefaultCamelContext.doStartOrResumeRoutes(DefaultCamelContext.java:3640) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.impl.DefaultCamelContext.doStartCamel(DefaultCamelContext.java:3492) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.impl.DefaultCamelContext.access$000(DefaultCamelContext.java:209) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.impl.DefaultCamelContext$2.call(DefaultCamelContext.java:3251) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.impl.DefaultCamelContext$2.call(DefaultCamelContext.java:3247) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.impl.DefaultCamelContext.doWithDefinedClassLoader(DefaultCamelContext.java:3270) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.impl.DefaultCamelContext.doStart(DefaultCamelContext.java:3247) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.support.ServiceSupport.start(ServiceSupport.java:61) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.impl.DefaultCamelContext.start(DefaultCamelContext.java:3163) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.spring.SpringCamelContext.start(SpringCamelContext.java:133) ~[camel-spring-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	... 24 common frames omitted
Caused by: java.lang.IllegalArgumentException: No enum constant com.amazonaws.regions.Regions.us-west-1
	at java.lang.Enum.valueOf(Enum.java:238) ~[na:1.8.0_191]
	at com.amazonaws.regions.Regions.valueOf(Regions.java:26) ~[aws-java-sdk-core-1.11.269.jar!/:na]
	at org.apache.camel.component.aws.sqs.SqsEndpoint.createClient(SqsEndpoint.java:314) ~[camel-aws-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.component.aws.sqs.SqsEndpoint.getClient(SqsEndpoint.java:274) ~[camel-aws-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.component.aws.sqs.SqsEndpoint.doStart(SqsEndpoint.java:117) ~[camel-aws-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.support.ServiceSupport.start(ServiceSupport.java:61) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.util.ServiceHelper.startService(ServiceHelper.java:75) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at io.syndesis.integration.component.proxy.ComponentProxyEndpoint.doStart(ComponentProxyEndpoint.java:152) ~[integration-component-proxy-1.7-SNAPSHOT.jar!/:1.7-SNAPSHOT]
	at org.apache.camel.support.ServiceSupport.start(ServiceSupport.java:61) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.util.ServiceHelper.startService(ServiceHelper.java:75) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.impl.RouteService.doWarmUp(RouteService.java:157) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	at org.apache.camel.impl.RouteService.warmUp(RouteService.java:145) ~[camel-core-2.21.0.fuse-740005.jar!/:2.21.0.fuse-740005]
	... 36 common frames omitted

```

It works when I manually change the region in the ARN from `us-west-1` to `US_WEST_1`

--- a/app/extension/bom/pom.xml
+++ b/app/extension/bom/pom.xml
@@ -28,7 +28,7 @@
 
   <properties>
     <spring-boot.version>1.5.13.RELEASE</spring-boot.version>
-    <camel.version>2.21.0.fuse-740005</camel.version>
+    <camel.version>2.21.0.fuse-740007</camel.version>
   </properties>
 
     <!-- Metadata need to publish to central -->

--- a/app/integration/bom-camel-k/pom.xml
+++ b/app/integration/bom-camel-k/pom.xml
@@ -27,7 +27,7 @@
   <packaging>pom</packaging>
 
   <properties>
-    <camel.version>2.21.0.fuse-740005</camel.version>
+    <camel.version>2.21.0.fuse-740007</camel.version>
     <atlasmap.version>1.39.5</atlasmap.version>
     <jackson.version>2.8.11</jackson.version>
     <jackson.databind.version>2.8.11.3</jackson.databind.version>

--- a/app/integration/bom/pom.xml
+++ b/app/integration/bom/pom.xml
@@ -28,7 +28,7 @@
 
   <properties>
     <spring-boot.version>1.5.13.RELEASE</spring-boot.version>
-    <camel.version>2.21.0.fuse-740005</camel.version>
+    <camel.version>2.21.0.fuse-740007</camel.version>
     <atlasmap.version>1.39.6</atlasmap.version>
     <jackson.databind.version>2.8.11.3</jackson.databind.version>
   </properties>

--- a/app/pom.xml
+++ b/app/pom.xml
@@ -75,7 +75,7 @@
     <commons.compress.version>1.18</commons.compress.version>
 
     <!-- Global camel version used everywhere -->
-    <camel.version>2.21.0.fuse-740005</camel.version>
+    <camel.version>2.21.0.fuse-740007</camel.version>
     <camel-k-runtime.version>0.3.1</camel-k-runtime.version>
 
     <!-- Don't fork based on cores, doesn't work nicely in the cloud -->

